lamin_cli/compute/modal.py

```diff
@@ -0,0 +1,167 @@
+import os
+import subprocess
+import sys
+import threading
+from pathlib import Path
+
+import lamindb_setup as ln_setup
+import modal
+
+
+def run_script(path: Path):
+    """Takes a path to a script for running it as a function through Modal."""
+    result = {"success": False, "output": "", "error": ""}
+
+    def stream_output(stream, capture_list):
```

Comment:
Can we type this? Might want to underscore prefix this to make it clearer that this is an internal function inside another one.

Reply:
I was wondering whether this function is necessary in the way it exists, but didn't get this far yet.